ansible/roles/ooni-measurements/tasks/main.yml

```diff
@@ -1,19 +1,29 @@
 ---
+- template: src=ngx-oomsm-web dest=/etc/nginx/sites-enabled/oomsm-web
+  notify: reload nginx
 
-- name: Checkout "{{ ooni_measurements_repo }}" repository
-  git: >
-    depth=1
-    dest="{{ ooni_measurements_path }}"
-    repo="{{ ooni_measurements_repo }}"
-    force=yes
-    version="{{ ooni_measurements_repo_version }}"
+- name: docker network for measurements
+  docker_network:
+    name: msm
+    driver_options:
+      com.docker.network.bridge.name: brmsm
+    ipam_options:
+      subnet: 172.25.232.0/24
+      gateway: 172.25.232.1
 
-- name: Template secrets
-  template: >
-    src="{{ ooni_measurements_secrets }}"
-    dest="{{ ooni_measurements_secrets_path }}"
-
-- name: Build ooni-measurements production target
-  make:
-    chdir: "{{ ooni_measurements_path }}"
-    target: production
+- name: ooni-measurements webservice
+  docker_container:
+    image: openobservatory/ooni-measurements:{{ oomsm_tag }}
+    name: oomsm-web
+    hostname: oomsm-web
+    networks: [{name: msm, ipv4_address: '{{ oomsm_backend_ipv4 }}'}]
+    purge_networks: true
+    env:
+      APP_ENV: production
+      PRODUCTION: 'TRUE' # FIXME: is it ever used?
```

Comment:
On the other hand https://measurements-beta.ooni.io/files/by_date/whalambalamba does not print exception to the webpage so, probably, debug logging is disabled by some of these variables :)